jcommune-model/src/main/resources/org/jtalks/jcommune/model/entity/JCUser.hbm.xml

```diff
@@ -65,7 +65,7 @@
                 type="org.joda.time.contrib.hibernate.PersistentDateTime" not-null="false"/>
       <property column="POST_COUNT" name="postCount"/>
 
-      <set name="contacts" cascade="all-delete-orphan" inverse="true">
+      <set name="contacts" cascade="all-delete-orphan" inverse="true" fetch="join">
```

Comment:
Yes, you're right. It was needed for plugins, when created copy of JCUser. I did research and made conclusion that we don't need contacts in this scope, so I remove copying of contacts and added javadocs with explanations.